solutions/pandas_tests/Makefile

```diff
@@ -0,0 +1,54 @@
+# Some tests in files listed in selective_tests fail/ error out on execution.
+# These failures and error are caused by one of below:
+# 1. pypaper linux specific errors, documented
+# here - https://pyperclip.readthedocs.io/en/latest/#not-implemented-error
+# 2. s3fs error (which is currently an optional file handling mechanism in pandas)
+# https://bleepcoder.com/evalml/557744402/unit-test-fails-after-upgrading-to-pandas-1-0-0
+#
+# Make 'run-all' ignores the expected fail/error, and test the other tests in these files.
+
+TOP = $(abspath ../..)
+include $(TOP)/defs.mak
+
+APPBUILDER=$(TOP)/scripts/appbuilder
+TEST = pandas/tests/arrays/string_/test_string.py
+PYTEST_INI_FILE = pytest.ini
+
+OPTS = --max-affinity-cpus=4
+
+ifdef STRACE
+OPTS += --strace
+endif
+
+all: rootfs cpiofs
+
+appdir:
+	$(APPBUILDER) Dockerfile
+	cp $(PYTEST_INI_FILE) appdir/
+
+rootfs: appdir
+	cat selective_tests | while read line; do rm -f appdir/"$$line"; done
```

Comment:
I would recommend maintaining a lists of all, passed, and failed tests (as opposed the selective_tests the PR adds now), which is easier to maintain.

Reply:
Created a tests.passed list (is not used anywhere in Makefile/ code), should I keep it? The list test.partially_passed holds the python files with tests that fail (22/584).

Reply:
For the background, @mingweishih is trying to maintain consistence with other test suites, such as libcxx, which has test.passed, tests.all etc. And there is a run_test_locally script that automatically updates the lists. If there is no way to auto-update the tests.passed file, there is no need to check it in the repo.